main.go

```diff
@@ -20,24 +20,21 @@ import (
 	"flag"
 	"os"
 
-	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
-	// to ensure that exec-entrypoint and run can make use of them.
```

Comment:
Just curious: is this a known limitation of goio to drop such comments from the imports? This might be an issue in some cases, I guess. Fortunately, it didn't drop the `+kubebuilder:scaffold:imports` marker comment.